lib/conversions.js

```diff
@@ -0,0 +1,14 @@
+function toDecimal({ decimal = null, fractional = null }) {
+  if (decimal !== null) return decimal;
```

Comment:
Would we be able to use this style over one liners:

```
if (test) {
  return false;
}
```

Reply:
fine by me